coalib/misc/Exceptions.py

```diff
@@ -1,15 +1,11 @@
-from pyprint.NullPrinter import NullPrinter
-
-from coalib.misc import Constants
-from coalib.output.printers.LogPrinter import LogPrinter
-
 from pkg_resources import VersionConflict
 
+from coalib.misc import Constants
+import coalib.output.printers.LogHelper as LogHelper
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 
-def get_exitcode(exception, log_printer=None):
-    log_printer = (LogPrinter(NullPrinter()) if log_printer is None
-                   else log_printer)
 
+def get_exitcode(exception):
```

Comment:
hm not sure, it's about exceptions, so... actually let's consider that after the PR :+1: imo it's fine for now